projects/element-ng/datepicker/date-time-helper.ts

```diff
@@ -118,16 +118,23 @@ export const getFirstDayInMonth = (year: number, month: number): Date =>
  */
 export const getWeekOfYear = (date: Date, weekStart: WeekStart): number => {
   // Algorithm rewritten from C# example given at http://en.wikipedia.org/wiki/Talk:ISO_week_date
```

Comment:
Ok instead, maybe a comment here
```suggestion
  // Algorithm rewritten from C# example given at http://en.wikipedia.org/wiki/Talk:ISO_week_date
  // The algorithm is also adjusted for Sunday/Saturday starts
```